viz/charts/linkerd-viz/Chart.yaml

```diff
@@ -11,7 +11,7 @@ kubeVersion: ">=1.21.0-0"
 name: "linkerd-viz"
 sources:
 - https://github.com/linkerd/linkerd2/
-version: 30.9.1-edge
+version: 30.9.2-edge
```

Comment:
same here, this should be kept out of the change

Reply:
Reverted back the change as well as on the README file

Reply:
I have also reverted the multicluster helm chart version.